sdk/storage/azure-storage-file-share/tests/test_file_async.py

```diff
@@ -2345,27 +2520,31 @@ async def test_azure_named_key_credential_access(self, storage_account_name, sto
         properties = await file_client.get_file_properties()
 
         # Assert
-        self.assertIsNotNone(properties)
+        assert properties is not None
 
     @FileSharePreparer()
-    def test_account_sas_raises_if_sas_already_in_uri(self, storage_account_name, storage_account_key):
-        with self.assertRaises(ValueError):
+    @recorded_by_proxy_async
```

Comment:
I assume the recording for this is empty? If so, you can remove the record_by_proxy since this doesn't make any network calls. This applies to sync as well.